src/elements-experimental/core/timetable/access-leg-helper.ts

```diff
@@ -178,15 +185,25 @@ export function getDepartureArrivalTimeAttribute(
     ? {
         text: i18nWalkingDistanceDeparture[currentLanguage],
         duration: departureWalk,
-        icon: 'walk-small',
+        icon: a11yFootpath ? 'wheelchair-small' : 'walk-small',
       }
     : null;
+  const extendedA11yFirstLeg =
```

Comment:
I wouldnt introduce a new const for this case. You can extend the function `getFirstExtendedLegAttribute`
Also make sure that the user gets the right text.

Reply:
Suggestion applied. 
Currently, only the aria-text for the visible icon is shown. A11y checks are running.